bim2sim/elements/bps_elements.py

```diff
@@ -526,15 +525,50 @@ def _get_persons(self, name):
         default=0.5,
     )
     machines = attribute.Attribute(
+        description="Specific internal gains through machines, if taken from"
+                    " IFC property set a division by thermal zone area is"
+                    " needed.",
         default_ps=("Pset_SpaceThermalLoad", "EquipmentSensible"),
-        unit=ureg.watt,
+        ifc_postprocessing=_area_specific_post_processing,
+        unit=ureg.W / (ureg.meter ** 2),
     )
+
+    def _calc_lighting_power(self, name) -> float:
+        if self.use_maintained_illuminance:
+            return self.maintained_illuminance / self.lighting_efficiency_lumen
+        else:
+            return self.fixed_lighting_power
+
     lighting_power = attribute.Attribute(
+        description="Specific lighting power, if taken from IFC property set"
+                    "a division by thermal zone area is needed.",
         default_ps=("Pset_SpaceThermalLoad", "Lighting"),
-        unit=ureg.W,
+        ifc_postprocessing=_area_specific_post_processing,
+        functions=[_calc_lighting_power],
+        unit=ureg.W / (ureg.meter ** 2),
+
+    )
+    fixed_lighting_power = attribute.Attribute(
+        description="Specific fixed electrical power for lighting in Lx. This "
+                    "value is taken from SIA 2024.",
+        unit=ureg.W / (ureg.meter ** 2)
+    )
+    maintained_illuminance = attribute.Attribute(
+        description="Maintained illuminance value for lighting. This value is"
+                    " taken from SIA 2024.",
+        unit=ureg.lumen / (ureg.meter ** 2)
+    )
+    use_maintained_illuminance = attribute.Attribute(
```

Comment:
Don't know what to say. The formular is stated here
```
    def _calc_lighting_power(self, name) -> float:
        if self.use_maintained_illuminance:
            return self.maintained_illuminance / self.lighting_efficiency_lumen
        else:
            return self.fixed_lighting_power
```

* `maintained_illuminance` is lm/m² that are required in the room.
* `lighting_efficiency_lumen` is lm/W_el. So how many lumen do you get from the lighting system
* so by diving them, you get W_el/m² that are required.

The only simplifications are:
* The `maintained_illuminance` is measured at desk level, while the `lighting_efficiency_lumen` is specified for the lighting system regardless of its position. We assume that all light reaches the desk, which is not entirely accurate.
* We could implement a factor assuming that only 70% of the light reaches the desk, but I would prefer to discuss this with the TEASER group first.

--> This simplification means we would actually need more lighting power, which would increase the internal loads. Generally, this leads to an overestimation of heating demand and an underestimation of cooling demand. However, this is still much more accurate than the previous calculations, which assumed halogen lamps throughout, resulting in a significant overestimation of internal loads.

Reply:
I cannot find any related lighting efficiency in lm/W_el in SIA 2024 that matches your inputs, so I guess you got this parameter from literature?

Which SIA 2024 version are you referring to here for maintained illuminance and fixed_lighting_power? 
I checked 2006 and 2021, and e.g., for Hotel rooms, I don't find a fixed lighting power of 2.9W/m2, and no maintained_illuminance of 200lux for hotel rooms. 

For maintained_illuminance, I checked some parameters, and I found mismatches/inconsistencies for classroom (school), main hall, retail stores, data center, Exhibition room, ... . 
I don't know which SIA you use, but SIA 2024 refers to SIA 387/4 (not sure which version), which names "Beleuchtungsstärke in lx" and Referenzbeleuchtungsstärke in lx". The parameters in the data set sometimes switch between those two values (or, if listed above, use none of these two values). 

There is a missing value in maintained_illuminance for spectator area (theater and event venues). 

Additionally, if you check SIA 2024 (2021), most of the fixed_lighting_power parameters (unless there are erros) refer to the parameters for existing buildings ("Bestandsdaten"), but using the standard values, or even "Zielwerte" would make even the use of the fixed_lighting_power parameter more applicable.

Reply:
Let me adress these step by step:

* data comes from DIN EN V 18599-10 2018-09 & SIA 2024 from 2015-10, I added some central documentation about the enrichment and where the data comes from via commit 7fba664260889ea8e799574359d2440da7473c10
* For hotel rooms and some usages, we used the Data from 18599 (200 lx), because the data seemed more reasonable. The 50 lx for a hotel room (SIA 2024) seemed a bit less. This definitely can be discussed. If you want to make changes here feel free to adjust the templates. 
* The missing value for "Spectator area (theater and event venues)" is a mistake. DIN V EN 18599 suggests a value of 200 lx, which I a added with commit 510083ab8cc26382b5183aad4e656d8f6dce3038
* Feel free to add the "Zielwerte" from SIA 2024. This can be addressed in a separate issue.

Reply:
Thank you for addressing these issues.

Please add the link to the [useconditions.py](https://github.com/RWTH-EBC/TEASER/blob/main/teaser/logic/buildingobjects/useconditions.py ) to the documentation, as it includes the units of the dataset. 

Please update the documentation of the [maintained_illuminance]( https://github.com/RWTH-EBC/TEASER/blob/main/teaser/logic/buildingobjects/useconditions.py#L139) according to what you just mentioned (i.e., parameters are not just from SIA, but also from DIN EN 18599, at least partially).

Reply:
adressed with commit [1f45cf4](https://github.com/BIM2SIM/bim2sim/pull/741/commits/1f45cf49a0b5d468907f8093966cc83e26c490ba) in bim2sim and [PR 792](https://github.com/RWTH-EBC/TEASER/pull/792)in teaser